### ISSUE TITLE
[Sourcing][Accessibility] fdp-select list box arrow button needs aria-label

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#9535]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
The disabled user cannot understand what is the icon for on the screen, hence we need an aria-label for fdp-select.
This is reported by Deutsche telekom customer

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
angular 15
core / platform : 0.39.1-rc-5

#### If this is a bug, please provide steps for reproducing it.
This is replicable on https://sap.github.io/fundamental-ngx/#/core/select via jaws or voiceover , the arrow button needs an aria-label

**Comment by fkolar** [comment]
@I543348, When checking this [reference implementation on SAPUI5 side ](https://sapui5.hana.ondemand.com/#/entity/sap.m.Select/sample/sap.m.sample.Select), they have `ariba-hidden=true` for this specific usecase. 


Do you mean this should be added to the arrow button?

**Comment by I543348** [comment]
Yes @fkolar , we need aria-label for this use case

**Comment by fkolar** [comment]
Closing, with keyboard and focus you never move on the arrow button and it makes less sense to have two UI element to expand the option this is why its one piece like here: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select

**Comment by I543348** [comment]
After checking this issue in application , found that focus never goes to "arrow button" as its one-piece so screenreader reads out the whole select as list box.

I wasnt able to check this on current NGX site , this can be closed.
Thanks @fkolar
